src/gateway/routers/axelar/Router.sol

```diff
@@ -73,7 +59,10 @@ contract AxelarRouter is Auth {
         string calldata sourceChain,
         string calldata sourceAddress,
         bytes calldata payload
-    ) public onlyCentrifugeChainOrigin(sourceChain, sourceAddress) {
```

Comment:
Removing modifier as there's little value in this if it's not reused, and it makes the code easier to read.